department-of-veterans-affairs/vets-website
# Add Express Care requests to past appointments list

## Description
This PR
- Fetches requests along with past appointments for the new homepage
- Filters EC requests so that fulfilled show up anytime, but cancelled and pending requests only show up when they're more than 2 days old.

How to test
- Turn on va_online_scheduling_homepage_refresh flag
- Go to http://localhost:3001/health-care/schedule-view-va-appointments/appointments/past
- See that EC requests show up in past appts list

## Testing done
Local and unit testing

## Screenshots
![Screen Shot 2021-02-10 at 10 47 56 AM](https://user-images.githubusercontent.com/634932/107533949-73a9ca00-6b8d-11eb-861f-539d6847d816.png)

## Acceptance criteria
- [ ] EC requests show up on past appt list under correct conditions

## Definition of done
